cstruct.opam

```diff
@@ -12,7 +12,7 @@ build: [
   ["jbuilder" "subst" "-p" name "--name" name] {pinned}
   ["jbuilder" "build" "-p" name "-j" jobs]
 ]
-build-depends: ["jbuilder" "runtest" "-p" name]
+build-test: ["jbuilder" "runtest" "-p" name]
```

Comment:
Sure, I'll fix it once the actual code is sorted. 

To address some of your queries about more reviews and regressions: it's not actually a regression as we were never actually testing some of this functionality before.  Breaking out ppx into a separate package  is a new feature that is causing the need for separating the test dependencies as well -- a feature I implemented upon a request from you in fact :-)

The original PR in question had 3 reviews, so the decision then was to prioritise safety over performance.  The decision now is to restore performance but with careful review to ensure that there is no safety regression.

Iow, I see a healthy process in place with features being added, reviews being conducted, and increasing test coverage over a more modular codebase that supports separate dependencies across ppx, async, lwt, unix and mirage.  Thanks for taking the time to submit these ancillary fixes as part of this process!